src/components/IncidentInfo/IncidentInfo.js

```diff
@@ -19,9 +19,10 @@ const IncidentInfo = ({ id }) => {
   return (
     <section className="incident-info-container">
```

Comment:
Once the weather component is added, we might want to use flexbox on this container to space things evenly.